curl_fuzzer.h

```diff
@@ -78,6 +78,187 @@
 #define TLV_TYPE_CONNECT_ONLY			50
 #define TLV_TYPE_HSTS				51
 #define TLV_TYPE_HTTPPOSTBODY			52
+#define TLV_TYPE_PROXY				53
+#define TLV_TYPE_PROXYTYPE			54
+
+#define TLV_TYPE_PROXYUSERPWD 100
```

Comment:
I have a mild preference for these values to be block aligned to the right like the others, but I'm not going to be militant about it.